<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qt_Opengl_FrameworkClass</class>
 <widget class="QMainWindow" name="Qt_Opengl_FrameworkClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt_Opengl_Framework</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>581</width>
      <height>541</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuColor">
    <property name="title">
     <string>Color</string>
    </property>
    <addaction name="actionGray"/>
    <addaction name="actionUniform_Quantization"/>
    <addaction name="actionPopulosity"/>
   </widget>
   <widget class="QMenu" name="menuDithering">
    <property name="title">
     <string>Dithering</string>
    </property>
    <addaction name="actionNaive_Threshold_Dithering"/>
    <addaction name="actionBrightness_Preserving_Threshold_Dithering"/>
    <addaction name="actionRandom"/>
    <addaction name="actionClustered"/>
    <addaction name="actionFloyd_Steinberg"/>
    <addaction name="actionColor_Floyd_Steinberg"/>
   </widget>
   <widget class="QMenu" name="menuFiltering">
    <property name="title">
     <string>Filtering</string>
    </property>
    <addaction name="actionBox"/>
    <addaction name="actionBartlett"/>
    <addaction name="actionGaussian"/>
    <addaction name="actionArbitrary_Size_Gaussian"/>
    <addaction name="actionEdge"/>
    <addaction name="actionEnhance"/>
   </widget>
   <widget class="QMenu" name="menuResize">
    <property name="title">
     <string>Size&amp;&amp;Rotate</string>
    </property>
    <addaction name="actionHalf"/>
    <addaction name="actionDouble"/>
    <addaction name="actionArbitrary_Uniform_Scale"/>
    <addaction name="actionRotate"/>
   </widget>
   <widget class="QMenu" name="menuComposing">
    <property name="title">
     <string>Composing</string>
    </property>
    <addaction name="actionOver"/>
    <addaction name="actionIn"/>
    <addaction name="actionOut"/>
    <addaction name="actionAtop"/>
    <addaction name="actionXor"/>
   </widget>
   <widget class="QMenu" name="menuNPR">
    <property name="title">
     <string>Bouns</string>
    </property>
    <addaction name="actionNPR"/>
   </widget>
   <widget class="QMenu" name="menuPattern">
    <property name="title">
     <string>Pattern</string>
    </property>
    <addaction name="actionFind_Pattern"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuColor"/>
   <addaction name="menuDithering"/>
   <addaction name="menuFiltering"/>
   <addaction name="menuResize"/>
   <addaction name="menuComposing"/>
   <addaction name="menuNPR"/>
   <addaction name="menuPattern"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionGray">
   <property name="text">
    <string>Gray</string>
   </property>
  </action>
  <action name="actionUniform_Quantization">
   <property name="text">
    <string>Uniform Quantization</string>
   </property>
  </action>
  <action name="actionPopulosity">
   <property name="text">
    <string>Populosity</string>
   </property>
  </action>
  <action name="actionNaive_Threshold_Dithering">
   <property name="text">
    <string>Naive Threshold Dithering</string>
   </property>
  </action>
  <action name="actionBrightness_Preserving_Threshold_Dithering">
   <property name="text">
    <string>Brightness Preserving Threshold</string>
   </property>
  </action>
  <action name="actionRandom">
   <property name="text">
    <string>Random </string>
   </property>
  </action>
  <action name="actionClustered">
   <property name="text">
    <string>Clustered </string>
   </property>
  </action>
  <action name="actionFloyd_Steinberg">
   <property name="text">
    <string>Floyd-Steinberg </string>
   </property>
  </action>
  <action name="actionColor_Floyd_Steinberg">
   <property name="text">
    <string>Color Floyd-Steinberg </string>
   </property>
  </action>
  <action name="actionBox">
   <property name="text">
    <string>Box</string>
   </property>
  </action>
  <action name="actionBartlett">
   <property name="text">
    <string>Bartlett</string>
   </property>
  </action>
  <action name="actionGaussian">
   <property name="text">
    <string>Gaussian </string>
   </property>
  </action>
  <action name="actionArbitrary_Size_Gaussian">
   <property name="text">
    <string>Arbitrary-Size Gaussian </string>
   </property>
  </action>
  <action name="actionHalf">
   <property name="text">
    <string>Half </string>
   </property>
  </action>
  <action name="actionDouble">
   <property name="text">
    <string>Double </string>
   </property>
  </action>
  <action name="actionArbitrary_Uniform_Scale">
   <property name="text">
    <string>Arbitrary Uniform Scale</string>
   </property>
  </action>
  <action name="actionEdge">
   <property name="text">
    <string>Edge</string>
   </property>
  </action>
  <action name="actionEnhance">
   <property name="text">
    <string>Enhance</string>
   </property>
  </action>
  <action name="actionRotate">
   <property name="text">
    <string>Rotate</string>
   </property>
  </action>
  <action name="actionResample_src">
   <property name="text">
    <string>resample_src</string>
   </property>
  </action>
  <action name="actionOver">
   <property name="text">
    <string>Over</string>
   </property>
  </action>
  <action name="actionIn">
   <property name="text">
    <string>In</string>
   </property>
  </action>
  <action name="actionOut">
   <property name="text">
    <string>Out</string>
   </property>
  </action>
  <action name="actionAtop">
   <property name="text">
    <string>Atop</string>
   </property>
  </action>
  <action name="actionXor">
   <property name="text">
    <string>Xor</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionNPR">
   <property name="text">
    <string>NPR</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionFind_Pattern">
   <property name="text">
    <string>Find Pattern</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qt_opengl_framework.qrc"/>
 </resources>
 <connections/>
</ui>
